soweego/linker/classify.py

```diff
@@ -127,9 +127,7 @@ def execute(catalog, entity, model, name_rule, threshold, dir_io):
         wd_generator = workflow.preprocess_wikidata(
             'classification', wd_reader)
 
-        all_feature_vectors = []
-        all_wd_chunks = []
-        all_target_chunks = []
+        all_feature_vectors = all_wd_chunks = all_target_chunks = None
```

Comment:
This is potentially risky, to be 100% sure, I suggest:
```suggestion
        all_feature_vectors, all_wd_chunks, all_target_chunks = None, None, None
```